music.php

```diff
@@ -112,4 +117,35 @@ function tree($dir, $depth) {
 		else
 			return false;
 	}
+
+	function return_zip($name, $paths, $flat){
+		$windows = substr(php_uname(), 0, 7) == 'Windows';
+		$temp_path = tempnam(sys_get_temp_dir(), "mfp_");
+		$temp_file = fopen($temp_path, "w");
+		if ($windows) {
+			foreach($paths as $path) fwrite($temp_file, ($flat ? './' : ''). $path. "\r\n");

```

Comment:
`./` works like that? I read it on stackoverflow somewhere, but I can't really see how it would work. Anyway, I'll test this on a Windows machine soon.